src/mame/edevices/fantland.cpp

```diff
@@ -1374,7 +1431,7 @@ ROM_START( borntofiv )
 	ROMX_LOAD( "24.bin", 0x060002, 0x10000, CRC(82d5db61) SHA1(01e30fe48ec486b8cf4db1d137daa5104099bc21), ROM_SKIP(2) )
 	ROMX_LOAD( "40.bin", 0x060001, 0x10000, CRC(707a6ddd) SHA1(30ae010bb2bd41b847790d1612a0702bc0c8b646), ROM_SKIP(2) )
 	ROMX_LOAD( "56.bin", 0x060000, 0x10000, CRC(8e84cedf) SHA1(8e6578803fe34083d74922112fd8b76cce9d393f), ROM_SKIP(2) )
-	ROMX_LOAD( "25.bin", 0x090002, 0x10000, BAD_DUMP CRC(500d4d3b) SHA1(54a448e5e784633513b94fdd4ad79e66b5210108), ROM_SKIP(2) ) // Not from this set
+	ROMX_LOAD( "25.bin", 0x090002, 0x10000, CRC(500d4d3b) SHA1(54a448e5e784633513b94fdd4ad79e66b5210108), ROM_SKIP(2) )
```

Comment:
Just to confirm, did you find this set with a good ROM to redump?

Reply:
I dumped the ROM from the same PCB we tried the first time. It was giving variable checksums with Jordigahan’s programmer, but with a little pin cleanup I reads OK with mine.